### PR TITLE
Removes the Syringe Gun.

### PR DESCRIPTION
**Pull request by Iamgoofball**
# WHY
1. Ranged chemical application is broken as fuck. It sucks balls, it's annoying to fight, and to balance it we have to nerf every single chemical into the ground to the point where it's not worth it to use the chems at all.
2. It's just another taser.
3. It opens up a lot of gamebreaking shit because it lets you skip the stunning part of a kill.

>just nerf the kill chems GOD

people will just find new ones and this will repeat forever and you know it

>how do i poison people now???

put it in their food and in their pills or stun them with some other method

### CODE REVIEW

**Comment by PKPenguin321** [comment]
At first I instinctively thought that this was a salty coder nerf but the reasoning actually makes sense, I agree with you and hopefully if this is removed we can have some of the common chems be slightly more powerful again. Did you remove the rapid syringe gun from science, too?

**Comment by Iamgoofball** [comment]
Yeah, it's removed along with the pocket syringe gun traitors can order.

**Comment by optimumtact** [comment]
if you nerf this people will just move to smoke or foam application

your fix stinks of shortsighted salt coding because you died to a syringe gun

**Comment by Ergovisavi** [comment]
I'm all for removing the syringe gun in its current incarnation but would prefer the syringe gun simply only accept hand picked reagents, IE: Morphine ammo darts, etc.

If I had to pick this or nothing, though, fuck the syringe gun. It's just shit.

**Comment by CitrusGender** [comment]
is this week just shit on medical week?

**Comment by PKPenguin321** [comment]
oranges, those are not quite as instant or concentrated as the syringe gun. I think this case is unique enough to warrant this

**Comment by Iamgoofball** [comment]
Smoke and foam require a lot more effort to apply and you can easily counter those with:
1. Gas masks block smoke, and you can use internals to dodge it.
2. Foam actually works worse the more skin coverage you have, and only really applies max reagents for the first tick if you're standing on the origin tile, it has math shit to reduce chems per tick and the further out it's spread

**Comment by optimumtact** [comment]
I don't agree, this solves none of the issues with chemical death mixes, it does nothing than move the problem about a bit

**Comment by optimumtact** [comment]
You can easily block a syringe gun with thick clothing which can be found fairly easily

**Comment by Iamgoofball** [comment]
Thick clothing slows you down and no one will take dropping their base movement makinng them vulnerable to every other attack method jusst to block syringes.

**Comment by The-clown-known-as-Pepe** [comment]
@optimumtact smoke and foam are easily counterable by running from the grenade, syringe guns are point and click kills with no counter

**Comment by optimumtact** [comment]
except they have a counter, and yes goof that's called tradeoff, look the word up in a dictionary sometime

**Comment by The-clown-known-as-Pepe** [comment]
@optimumtact wearing a fire/biosuit at all times because you can get shot by a syringe gun someone can pull out and shoot you in half a second is a garbage counter, please don't try to deny it

**Comment by The-clown-known-as-Pepe** [comment]
smoke/foam give you visual and distinct sound indication that you're going to get fucked, it gives you a chance instead of dropping dead because you didn't wear a firesuit

**Comment by CitrusGender** [comment]
@The-clown-known-as-Pepe You can say that exact same thing about the Tazer Gun except that a fire/biosuit doesn't deflect it and you can fire it multiple times without reloading.

**Comment by The-clown-known-as-Pepe** [comment]
@CitrusGender they don't instakill you

**Comment by CitrusGender** [comment]
@The-clown-known-as-Pepe I mean, it takes like 3 seconds to put cuffs on somebody if you have them tazed on the ground. I've never really seen a fight continue on after the first opponent gets tazed unless they're running like adrenaline implants or they have the syndie implants (though, the syringe gun is never effective on nuke ops.)

**Comment by Bawhoppen** [comment]
Regardless of your stance, I see no reason to remove it from the code itself (The more admin only items, the better).

Anyways the concept of the syringe gun is really cool, and removing it would be an utter shame. Maybe remove it roundstart? (Not suggesting it should be available from R&D instead)

**Comment by The-clown-known-as-Pepe** [comment]
Let me put it this way

Do we want a weapon that can remove people from round if they don't counter it 24/7 in game?

**Comment by Ergovisavi** [comment]
The syringe gun has no true purpose for existing outside of basically instantly killing people.

* The syringe gun is used by zero medbay staff to actually do their jobs, because they work on people who WANT their care
* The syringe gun does not create interesting gameplay scenarios. Not once has the round been bettered by use of one, it is a tool simply for removing players from the round in one click with minimal consequences. At least bombs, you know, blow up the station and the fallout from them create interesting situations.
* Your only protection is wearing thick clothing, which always slows the player down, which is SUBSTANTIALLY UNFUN and reeks of metagaming, similar to disabling the plasma loop at the start of the round, which is a bannable offense
* Ammo is so laughably littered all around the station because it's not considered a piece of contraband
* It takes melee range, a delay, AND a warning message to inject FIVE UNITS of reagents into someone with a syringe. And a syringe gun does triple that, at range, instantly.

The syringe gun is a sterling example of completely garbage design.

**Comment by Bawhoppen** [comment]
don't try to tell me it's less interesting than a bullet

**Comment by Ergovisavi** [comment]
>don't try to tell me it's less interesting than a bullet

A bullet
* A projectile that interacts with the health system requiring you to seek aid and is not instantly lethal, causing you to interact with the health system to fix it. Usually used by security (Detective) for the sake of arresting people which can lead to deconversion (player continues to play), perma (player can potentially still play depending on other player interaction or the abilities they have).
* Unsurprisingly, gunfights exist as a result of bullets which are combat encounters that don't instantly end, see literally all of Nuke Ops
* Bullets normally come from one highly visible source on the station and most people don't have access to it
* If you're shot you can scream for help and now people start searching for you, sparking a manhunt

A syringe gun
* You're dead
* You couldn't cry for help
* Enjoy being a ghost

Yeah, I'd say the bullet is more interesting.

**Comment by CitrusGender** [comment]
@Ergovisavi It sounds like you have more of a problem with the stuff inside if the syringe gun than the actual syringe gun. People keep talking about death mixes, but the only one that I really know of is neurotoxin and beepsky smash, both of which are just stuns. I am all for tweaking those chemicals but to remove the gun is ridiculous since the gun itself is not the problem in this case. To be honest, the only thing I have ever been shot with is morphine and that takes forever to stun anyways. You also act like a taser doesn't do the exact same thing otherwise (assuming it's neurotoxin), which is certainly not true [though the stun is a little bit long].

**Comment by Iamgoofball** [comment]
@CitrusGender 
Please read the OP.

**Comment by KorPhaeron** [comment]
I miss the freeze

**Comment by Iamgoofball** [comment]
me too, i have to actually make PRs now

**Comment by ExcessiveUseOfCobblestone** [comment]
just make originally op chems have a different/lesser effect if injected wtf this is dumb.

**Comment by Hatfish** [comment]
don't remove it, the syringe gun has a legitimate use in holding back the greytide whenever they rush medbay storage.

**Comment by Repukan** [comment]
Why not make it an uplink exclusive item.

**Comment by MMMiracles** [comment]
If you want to bitch about ranged chemical unfairness, remove the dart shells instead of the syringe gun. At least with the syringe gun you have to manually load each shot, a combat/riot shotgun with dart shells are like a syringe gun on steroids and are absolutely devastating if someone puts the effort into it.

**Comment by Ergovisavi** [comment]
>Instead of

You meant "in addition to"

**Comment by MMMiracles** [comment]
Did I stutter?

**Comment by The-clown-known-as-Pepe** [comment]
@MMMiracles for dart shells you need a shotgun that you can only get from 3 places on station

Brig, cargo and barkeep

**Comment by MMMiracles** [comment]
So the shotgun is more commonly available than a syringe gun and you've only further proven my point.

**Comment by The-clown-known-as-Pepe** [comment]
@MMMiracles >COMMONLY AVAIBLE

yeah you can get them easily by just doing these things

1. raid brig for shotguns
2. get QM to buy shotguns for you
3. kill barkeep for his shotgun
😂 
instead of picking one of 2-3 syringe guns that spawn on every station in medbay

**Comment by MMMiracles** [comment]
You also seem to forget the improvised shotgun, not like any of your listed options are exactly difficult or the only options to get a shotgun. 

Dart shells are objectively superior and easier to get. This PR is dumb.

**Comment by The-clown-known-as-Pepe** [comment]
@MMMiracles 
>dart shells are easier to get

they take 2k metal to make in autolathe

syringes take 60 metal to make and are everywhere

improvised shotgun requires materials to make, if you don't have a modular reciever then you're screwed

shit arguments tbh

**Comment by XDTM** [comment]
We are kinda lacking in ways to give chems to a target; poisoning food is interesting, but nobody eats food, cryo requires your target to be already hurt, grenades are stopped by gasmasks or moving away, and apply a minimal amount. You have to be a traitor and buy a sleepypen, pretty much. So if i want to nonlethally kidnap someone as another antag i'm outta luck, whatever i do they have time to call for help on the radio.

**Comment by jughu** [comment]
But shotgun darts are 30 units and no react

**Comment by jughu** [comment]
Besides you can say if you dont have a modular receiver your screwed which you can also make in autolatjes

**Comment by The-clown-known-as-Pepe** [comment]
thought reciever was rnd thingy, huh

File: _maps/RandomRuins/LavaRuins/lavaland_biodome_beach.dmm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE 
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (RemieRichards) [inline _maps/RandomRuins/LavaRuins/lavaland_biodome_beach.dmm]: smells like fastdmm

**Comment by Artemiuz28** [comment]
If you're so fucking scared of getting syringed, then carry charcoal pills. They're available at chemical storage and increase metabolization of everything by 1u/cycle.

**Comment by jughu** [comment]
Try to take a pill when your shot with nuerotoxin

**Comment by jughu** [comment]
Or ill just replace the charcoal pills

File: _maps/RandomRuins/LavaRuins/lavaland_biodome_beach.dmm (hunk 1 above)
Comment (Iamgoofball) [reply _maps/RandomRuins/LavaRuins/lavaland_biodome_beach.dmm]: Didn't use it

**Comment by calzilla1** [comment]
Make race potions instant via melee. Other wise its a :-1:

**Comment by kevinz000** [comment]
dont remove the delivery method
balance the system.

**Comment by KingPhilipIII** [comment]
>Giving your own PR a thumbs up.

Disgraceful.

**Comment by Fox-McCloud** [comment]
Definitely agree with this. This will make people utilize foam and smoke more, no doubt, but those are still more risky to utilize (most of the time) than syringe guns.

I don't recommend removing them, entirely, but restricting them to traitor only would be an excellent start in my opinion.

At the end of the day, syringe guns will always be the go-to application delivery method because they're targeted, have little to no collateral, are ranged, and nearly instant. In most scenarios or situations, they're the optimal way to deliver "deathchems".

While you could do any number of things to balance it, at the end of the day, you still have a ranged insta-application weapon that doesn't even require a firing pin to operate; this alone is going to make it amongst the most attractive weapons in the game for those in/around science (and engineering, of course, since setting up and R&D lab is trivial).

Shotgun darts would probably also have to go/made traitor only; as soon as the syringe gun is removed, everyone will flock to making makeshift shotguns.

I think Goon got this one right; syringe guns and grenades are both traitor-only there. Chems are still very dangerous, of course, but using them on a mass scale willy-nilly isn't something you generally see by the crew; it's a traitor unique thing (and that's a good ideal, in my opinion).

**Comment by Thelolswat** [comment]
just carry charcoal tho

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm going to be goof here and go:

## EDIT INJECTION EFFECTS FOR ""OP"" CHEMS

**Comment by PraiseRatvar** [comment]
how about remove syringe guns from the map, rather than removing them from the code completely?

**Comment by Bluespace** [comment]
This is a fairly big buff to wizard so I'm actually okay with this going through.

Makes hulks harder to deal with but I suppose they can just be killed instead. 

And as someone pointed out, shotgun darts are better anyway.

**Comment by FuryMcFlurry** [comment]
Classic grudge removal. Syringe guns are obvious as fuck. Has a large inhand sprite and its clunky to reload.
I honestly think the syringe gun needs to have the same gun laws as the bartender shotgun. Dont take it out of medbay. Maybe reduce the syringe gun spawn to 1. 
if we are going the route of removing the syringe gun add them to a space ruin or something. 
End of rant.

**Comment by Artemiuz28** [comment]
No reasons to remove them.
 If chemist is antag, you would be dead anyway.
 If he's not, then you should admin-help for (job)banning him.
 Otherwise, you're a clown or assistant, who didn't get killed for no reasons.

**Comment by JJRcop** [comment]
What if the syringes embedded in you and injected their chems over time, automatically falling out when they're empty?

**Comment by Bawhoppen** [comment]
That sounds like it would be really cool actually

**Comment by Artemiuz28** [comment]
@JJRcop, this makes sense for patches, but not for syringes really.

**Comment by PopNotes** [comment]
_this is the guy people actually want for headmin_

**Comment by ma44** [comment]
>Implying that removing a item that people hate would make goof not as popular

**Comment by Hatfish** [comment]
> implying that removing an item that some rarely plays and a fraction of the playerbase hates would make goof not as popular

**Comment by lzimann** [comment]
Stale and all of that.

**Pull request closed without merging**